lib/web/mage/backend/tree-suggest.js

```diff
@@ -55,7 +55,8 @@ define([
         /**
          * @override
          */
-        init: function () {
+        init: function (el,options) {
+            init.call(this,el,options);
```

Comment:
Let's add spaces between params

```suggestion
        init: function (el, options) {
            init.call(this, el, options);
```
Perhaps JS Coding Style requires it
https://javascript.info/coding-style

Reply:
Hello, @andrewbess , I have implemented the suggestions you provided  on my pull request